app/models/ckeditor/picture.rb

```diff
@@ -1,5 +1,6 @@
 class Ckeditor::Picture < Ckeditor::Asset
   mount_uploader :data, CkeditorPictureUploader, mount_on: :data_file_name
+  self.inheritance_column = nil
```

Comment:
cần type cho cái gì vậy?

Reply:
cái upload ảnh của em nó bị lỗi nên em thêm để sửa đấy mà :3 
https://github.com/galetahub/ckeditor/issues/739
which tells Rails to ignore the `type` column with respect to STI.
Để thằng rails nó bỏ qua cái cột type ạ :3